content/en/chaos_engineering.md

```diff
@@ -8,7 +8,7 @@ category: concept
 Chaos Engineering or CE is the discipline of experimenting on a [distributed system](https://glossary.cncf.io/distributed_systems/) in production to build confidence in the system's capability to withstand turbulent and unexpected conditions.
 
 ## Problem it addresses
-People working in the field of [SRE](https://glossary.cncf.io/site_reliability_engineering/) and [DevOps](https://glossary.cncf.io/devops/) continuously look for new techniques to increase product resiliency and [reliability](https://glossary.cncf.io/reliability/). A system's ability to tolerate failures while ensuring adequate service quality is typically a software development requirement. But deep testing on a production system is not always possible due to the complexity of the architecture. High-frequency deployment of new features to the production environment can result in a high probability of downtime and a critical incident — with considerable consequences to the business.
+[SRE](https://glossary.cncf.io/site_reliability_engineering/) and [DevOps](https://glossary.cncf.io/devops/) engineers continuously look for new techniques to increase product resiliency and [reliability](https://glossary.cncf.io/reliability/). A system's ability to tolerate failures while ensuring adequate service quality is typically a software development requirement. But deep testing on a production system is not always possible due to the complexity of the architecture. High-frequency deployment of new features to the production environment can result in a high probability of downtime and a critical incident — with considerable consequences to the business.
```

Comment:
@CathPag "I don't really like to personify terms like SRE but especially DevOps" has been an older review comment from my side. I think while with SRE it might be more or less clear or common sense what a Site Reliability Engineer does and what it is. With DevOps there is no common sense what exactly it is, and if we can or should call people even DevOps Engineers. This is at least what I experienced. 

What about something like.

"SRE and DevOps practices focus on techniques to ..." or something like that.

Reply:
ok, fine for me